dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -126,16 +122,13 @@ export const Discussion = ({
 	}, [filters, commentPage, shortUrlId]);
 
 	useEffect(() => {
-		const orderByClosed = isClosedForComments ? 'oldest' : undefined;
```

Comment:
I've removed this as this is not working as expected. Ive created a ticket [here](https://github.com/orgs/guardian/projects/88?pane=issue&itemId=51173252) to look into this.